include/net/net_ip.h

```diff
@@ -52,6 +52,9 @@ enum net_ip_protocol {
 	IPPROTO_TCP = 6,
 	IPPROTO_UDP = 17,
 	IPPROTO_ICMPV6 = 58,
+	IPPROTO_TLS_1_0 = 256,
```

Comment:
Should we support TLS 1.0 at all, I though it was kind of obsolete already?

Reply:
Perhaps not anymore, as there is a suggested migration from TLS 1.0 to TLS 1.1 or higher before June 30, 2018, see https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_1.0